### PR TITLE
[WIP DO NOT MERGE] Simplify nht for the proposed new Heroku build and deployment process

### PR DESCRIPTION
**Pull request by kitkat119**
This PR deletes the nht tasks that will no longer be used in the proposed new build and deployment process. I have removed the following tasks:
- ship
- deploy
- scale
- float
- drydock
- destroy
- provision
- smoke (already deprecated)
- test-urls (already deprecated)

I have also removed lib/commit and lib/verify-cache-headers as these were used by the deploy task only. It also appeared that the test fixtures and mocks were not being used. 

The deploy-static task is currently only used by `dewey-template` and `n-service-worker`. Dewey is soon to be deprecated so this could probably be taken out of nht too at some point.

### CODE REVIEW

**Comment by kitkat119** [comment]
One thing I was wondering - do you think I could delete lib/logger and the associated ascii art? It is used in bin/n-heroku-tools though.

**Comment by adgad** [comment]
I reckon you can replace /lib/logger with either `n-logger` or maybe even `console.log/error?`.

Seems to just be logging the error out...

File: bin/n-heroku-tools.js [hunk 1]
```diff
@@ -22,20 +22,11 @@ const utils = {
...
 program.version(require('../package.json').version);
 
-require('../tasks/deploy')(program, utils);
 require('../tasks/configure')(program, utils);
-require('../tasks/scale')(program, utils);
-require('../tasks/provision')(program, utils);
-require('../tasks/destroy')(program, utils);
 require('../tasks/deploy-hashed-assets')(program, utils);
 require('../tasks/deploy-static')(program, utils);
 require('../tasks/run')(program, utils);
```
[the remark this answers is not in the document]
Comment (kitkat119) [reply bin/n-heroku-tools.js]: nht run `Runs the local app through the router`. It is in the Makefile in next-search-page and I presumed that was intentional so I kept it...?

[the remark this answers is not in the document]
Comment (adgad) [reply bin/n-heroku-tools.js]: Ah yes of course! Yeah I guess leave it for now - but feels like something else the shouldn't be in heroku tools! Maybe it should be in next-router?

**Pull request merged**